### PR TITLE
Improve some package descriptions

### PR DESCRIPTION
**Pull request by DanSGiesbrecht**
An examle of improvements:
- Remove `package` @ end of descriptions
- Convert `DIP8` to `DIP-8`
- Convert `QFN` to `QFN-24`

A note: this is not full coverage. This could be scripted, to get better coverage over the full library

### CODE REVIEW

**Comment by chschlue** [comment]
@DanSGiesbrecht ping

**Comment by DanSGiesbrecht** [comment]
@chschlue thanks for the ping. I'm not sure when I'll have time to look at this - work schedule has gotten incredibly heavy.

**Comment by chschlue** [comment]
Ok, I'll merge as-is. It's a big improvement already.

File: Interface_Expansion.dcm [hunk 1]
```diff
@@ -169,19 +169,19 @@ F http://www.ti.com/lit/ds/symlink/tca9534.pdf
...
 $ENDCMP
 #
 $CMP TCA9535DBT
-D 16-bit I/O expander, I2C and SMBus interface, interrupts, w/o pull-ups, SSOP-24 package
+D 16-bit I/O expander, I2C and SMBus interface, interrupts, w/o pull-ups, SSOP-24
 K ti parallel port
 F http://www.ti.com/lit/ds/symlink/tca9535.pdf
 $ENDCMP
 #
 $CMP TCA9535MRGER
```
[the remark this answers is not in the document]
Comment (evanshultz) [reply Interface_Expansion.dcm]: What is the issue. The default footprint, FP filter, and description all seem to match the DB package into in https://www.ti.com/lit/ds/symlink/tca9535.pdf. What am I missing?

File: Interface_Expansion.dcm [hunk 2]
```diff
@@ -193,13 +193,13 @@ F http://www.ti.com/lit/ds/symlink/tca9535.pdf
 $ENDCMP
 #
 $CMP TCA9535RGER
-D 16-bit I/O expander, I2C and SMBus interface, interrupts, w/o pull-ups, QFN-24 package
+D 16-bit I/O expander, I2C and SMBus interface, interrupts, w/o pull-ups, QFN-24
 K ti parallel port
 F http://www.ti.com/lit/ds/symlink/tca9535.pdf
 $ENDCMP
 #
 $CMP TCA9535RTWR
```
[the remark this answers is not in the document]
Comment (evanshultz) [reply Interface_Expansion.dcm]: You mean the default package has a 2.6mm square thermal pad and should be 2.45mm, right? It appears to me the RTW and RGE packages could probably use the same footprint, but we can make custom ones easily enough. Differences I see: pin (and pad) width and tiny difference in location of thermal vias. Anything else?

File: RF.dcm [hunk 3]
```diff
@@ -37,7 +37,7 @@ F http://www.ti.com/lit/ds/symlink/cc1200.pdf
 $ENDCMP
 #
 $CMP CC2500
```
[the remark this answers is not in the document]
Comment (evanshultz) [reply RF.dcm]: What is wrong here? I'm not seeing it.

**Comment by evanshultz** [comment]
@chschlue 
I'll help out to wrap up the issues you mention above so it gets done. But I have a couple questions which I've posted above.

**Comment by chschlue** [comment]
Guess I should have been more verbose back then.
I'll try to figure it out again.

File: Interface_Expansion.dcm (hunk 1 above)
[the remark this answers is not in the document]
Comment (chschlue) [reply Interface_Expansion.dcm]: My mistake.
I think I mixed up DB and PW.

File: Interface_Expansion.dcm (hunk 2 above)
[the remark this answers is not in the document]
Comment (chschlue) [reply Interface_Expansion.dcm]: That and 3d model height.
1.2mm, 1mm and 0.8mm for QFN, VQFN and WQFN, respectively.

File: RF.dcm (hunk 3 above)
[the remark this answers is not in the document]
Comment (chschlue) [reply RF.dcm]: The datasheet mentions RTK (page 86) and RGP (page 90) packages.
RGP is what's actually orderable, so it's fine (although a TI-specific FP is probably unnecessary)

**Comment by chschlue** [comment]
I think I got it now.

**Comment by evanshultz** [comment]
OK. So we really just need RGE and RTW footprints for TCA9535.

**Comment by chschlue** [comment]
Right.
RGE already exists as VQFN-24-1EP_4x4mm_P0.5mm_EP2.45x2.45mm and RTW is WQFN-24-1EP_4x4mm_P0.5mm_EP2.45x2.45mm, if I'm not mistaken.

**Comment by evanshultz** [comment]
@chschlue 
Thanks! Submitted https://github.com/KiCad/kicad-symbols/pull/2387.

**Comment by DanSGiesbrecht** [comment]
Thanks!

**Pull request merged**